haskell.nix/flake.nix

```diff
@@ -15,102 +15,127 @@
       flake = false;
     };
 
-    # if you want to run weeder:
-    ##haskell-nix-weeder = {
-    ##  url = "github:serokell/haskell-nix-weeder";
-    ##  flake = false;
-    ##};
+    flake-utils = { url = "github:numtide/flake-utils"; };
+
+    ## for weeder:
+    #haskell-nix-weeder = {
+    # url = "github:serokell/haskell-nix-weeder";
+    # flake = false;
+    #};
   };
 
-  outputs = { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, ... }:
+  outputs =
+    { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, flake-utils, ... }:
+    ## for weeder:
+    # { self, nixpkgs, haskell-nix, hackage-nix, stackage-nix, flake-utils, haskell-nix-weeder, ... }:
```

Comment:
Wouldn't it be less convoluted to just `@args` here, and then just have a snippet below using that?

Reply:
Yes, it would. Feel free to push this :)